docs/cdoc/qk-classical-register.rst

```diff
@@ -0,0 +1,37 @@
+===================
+QkClassicalRegister
+===================
+
+.. code-block:: c
+
+   typedef struct QkClassicalRegister QkClassicalRegister
+
+A common way to instantiate several bits at once is to create a register, such
+as by:
```

Comment:
I believe you should consider adding a quick description of what a quantum register is before jumping into an example of how to use it. Even if this structure already exist in Python, it is still good to provide the definition of the object in question.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/14355/commits/0e7a205080e6aa96331d57e4545498ad05e34e33